packages/node_modules/@ciscospark/widget-space/README.md

```diff
@@ -80,6 +81,50 @@ For the latest builds that are pulled from the head of the master branch:
   npm run build:package widget-space
   ```
 
+### NPM
+
+#### **NOTE: NPM INSTALL IS CURRENTLY IN BETA AND NOT SUPPORTED**
+
+To use the space widget within an existing React appliction, a developer can install the component directly from npm.
+
+1. Install the module via NPM
+
+  ```bash
+  npm install --save @ciscospark/widget-space
+  ```
+
+2. Add the following import statements
+
+  ```js
+  import SpaceWidget, {destinationTypes} from '@ciscospark/widget-space';
+  // Sass import required to style widgets
+  import '@ciscospark/widget-space/src/momentum.scss';
+  ```
+
+3. Declare and define the `WidgetSpace` properties
+
+  ```js
+    const spaceWidgetProps = {
+      accessToken: 'XXXXXXXXXXXXXX',  // Required
+      destinationType: destinationTypes.USERID,
```

Comment:
shouldnt we declare all the enum values?

Reply:
Done